var/spack/repos/builtin/packages/py-meson-python/package.py

```diff
@@ -25,12 +26,17 @@ class PyMesonPython(PythonPackage):
     version("0.9.0", sha256="6aa5a09ff5cce1c5308938ebbf3eab5529413c8677055ace1ac8c83d8a07b29d")
     version("0.8.1", sha256="442f1fa4cf5db50eea61170a6059c10fafd70977f5dbdf3441c106cd23b05e4c")
     version("0.8.0", sha256="b5c8a2727e6f6feaffc1db513244c9bdb5d0f689b45e24f4529b649b7710daf7")
-    version("0.7.0", sha256="9fcfa350f44ca80dd4f5f9c3d251725434acf9a07d9618f382e6cc4629dcbe84")
+    version(
+        "0.7.0",
+        sha256="9fcfa350f44ca80dd4f5f9c3d251725434acf9a07d9618f382e6cc4629dcbe84",
+        deprecated=True,
+    )
 
     depends_on("py-colorama", when="platform=windows", type=("build", "run"))
     depends_on("meson@0.63.3:", when="@0.11:", type=("build", "run"))
     depends_on("meson@0.63:", when="@0.9:0.10", type=("build", "run"))
     depends_on("meson@0.62:", type=("build", "run"))
+    depends_on("py-packaging@19:", when="@0.16:", type=("build", "run"))
```

Comment:
Why is `packaging` needed as a dependency? It's not in the upstream `pyproject.toml`.

Reply:
https://github.com/mesonbuild/meson-python/blob/0.16.0/pyproject.toml#L38

Reply:
Ah sorry, reading comprension problem on my side. I thought it was a `scikit-learn` update. Yes, for `meson-python` it is right.